src/lib/serviceRoutinesV2/postEntity.cpp

```diff
@@ -89,14 +89,24 @@ std::string postEntity
     if ((upcrsP->contextElementResponseVector[ix]->statusCode.code != SccOk) &&
         (upcrsP->contextElementResponseVector[ix]->statusCode.code != SccNone))
     {
+      if (upcrsP->contextElementResponseVector[ix]->statusCode.code == SccInvalidParameter)
+      {
+        // Ugly v1 -> v2 error conversion here :(
+        OrionError error(SccRequestEntityTooLarge, "NoResourcesAvailable", "No more than one geo-location attribute allowed");
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-orion/commit/a8d0650bc335b0e735f4c57a87096bbbede7507b